www/js/app.js

```diff
@@ -166,7 +165,24 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
-    
+ 
+    // define and store dark preference for matchMedia
+    var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+    // if 'prefers-color-scheme' is not supported in the browser
+    if (window.matchMedia('(prefers-color-scheme)').media === 'not all') {
+        var optionsToBeRemoved = document.querySelectorAll('.auto');
```

Comment:
Maybe there is a risk to match some other DOM elements that would use the same class name.
It would be safer to do that only on the drop-down list